packages/open-next/src/build.ts

```diff
@@ -677,15 +677,16 @@ async function createCacheAssets(monorepoRoot: string) {
 /* Server Helper Functions */
 /***************************/
 
-function compileCache() {
-  const outfile = path.join(options.outputDir, ".build", "cache.cjs");
+export function compileCache(format: "cjs" | "esm" = "cjs") {
+  const ext = format === "cjs" ? "cjs" : "mjs";
```

Comment:
Is it really necessary to create 2 variables here ?

Reply:
esbuild accepts `format: "esm"` which isn't a recognizable extension so we need this format -> ext mapping here.